crates/extensions/c8y_mapper_ext/src/tests.rs

```diff
@@ -99,7 +99,9 @@ async fn child_device_registration_mapping() {
     let cfg_dir = TempTedgeDir::new();
     let (mqtt, _http, _fs, mut timer, _ul, _dl) = spawn_c8y_mapper_actor(&cfg_dir, true).await;
 
-    timer.send(Timeout::new(())).await.unwrap(); // Complete sync phase so that alarm mapping starts
+    // Complete sync phase so that alarm mapping starts
+    trigger_timeout(&mut timer).await;
```

Comment:
This change has been required because the test are now using a `FakeServerBox` that handles message envelopes behind the scene but can no more send a response before a request has been actually sent.